Site/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "apidb",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
     "@types/cytoscape": "3.2.9",
     "@veupathdb/multi-blast": "^1.0.5",
     "@veupathdb/preferred-organisms": "^1.1.9",
-    "@veupathdb/wdk-client": "^0.5.6",
+    "@veupathdb/wdk-client": "^0.5.12",
```

Comment:
You'll also want to commit the new `yarn.lock` file. Speaking of which, what command did you use to upgrade `@veupathdb/wdk-client`?

Reply:
I used `yarn upgrade @veupathdb/wdk-client@latest`

Reply:
Great, then that command should have produced a new `yarn.lock`.